.cargo/config.toml

```diff
@@ -0,0 +1,3 @@
+[target.x86_64-unknown-linux-gnu]
+linker = "/usr/bin/clang"
+rustflags = ["-C", "link-arg=--ld-path=/usr/local/bin/mold"]
```

Comment:
Should this be:

```
rustflags = ["-C", "link-arg=-fuse-ld=/path/to/mold"]
```

Reply:
yea, it's deprecated, but is fine for now. esp considering stuff like this
https://github.com/rust-lang/rust/issues/94347